Rakefile

```diff
@@ -33,7 +35,7 @@ task rdoc: "website:rdoc_github_links"
 
 namespace :website do
   task :build do
-    sh "cd www; bundle exec jekyll build; cd .."
+    sh "cd www; BUNDLE_GEMFILE=Gemfile bundle exec jekyll build; cd .."
```

Comment:
Why doesn't `rvm` handle this? I would rather that this issue is fixed in `rvm` rather than here.

Reply:
It's quite complicated. rvm sets BUNDLE_GEMFILE envar pointing to current Gemfile. I'm don't know when it really happens and if it's done by rvm or bundler. And I have no systems right now to check it against other ruby versions manager.

However I see now that it does not fully solve the problem, as it requires `(cd www && bundle install)` on the first run anyway. WDYT about moving `github-pages` from `www/Gemfile` to `Gemfile` with `require: false`?

Reply:
`github-pages` pulls in a lot of gems, I feel like it could overwhelm anyone running `bundle install` on the top level, because they're now installing a lot of gems that are only necessary for building the website. I also want to have those separate, as it will make inspecting Shrine dependencies by looking at `Gemfile.lock` easier.

I'm ok with this change then, maybe we can just add a comment that `rvm` requires `BUNDLE_GEMFILE` to be set.

We could also run `bundle install` by default:

```rb
# rvm requires $BUNDLE_GEMFILE to be set
sh "cd www"
sh "BUNDLE_GEMFILE=Gemfile bundle install"
sh "BUNDLE_GEMFILE=Gemfile bundle install"
sh "cd ..."
```

Reply:
As I understand it, bundler itself will automatically find the current `Gemfile` in the current directory, or recursively stepping up to parent directories until it finds one. It is only necessary to set the `BUNDLE_GEMFILE` env var if you want bundler to use a gemfile other than the one it will find via it's default algorithm. 

Since you are setting the env var, if I understand what's going on, to the path it would find anyway using it's default algorithm -- It might work just fine without the env var set?

Reply:
But I also don't understand in general what's going on here, I gotta admit.